pkg/core/pipeline/pullrequest/main.go

```diff
@@ -70,7 +70,7 @@ func New(config *Config, sourceControlManager *scm.Scm) (PullRequest, error) {
 
 	p := PullRequest{
 		Title:  config.Title,
-		Config: config,
+		Config: *config,
```

Comment:
I'm wondering how this change could help? 

The type passed to the object was already a  `*Config`(e.g. a pointer) so the calling method on the `PullRequest` were already able to change its pointer reference.

Did I miss something?

Reply:
That was my understanding as well but it doesn't seem to work. I think that part would benefit from so refactoring. But I first needs to master the debugger :p

Reply:
Given this consideration + the fact that the updatecli compiled from this PR stills creates PR with the litteral `{{ source ...}}`, do you mind removing this commit, so we can proceed on the rest of the code changes (eg fixing #484) ?


Many thanks for taking time for this!

Reply:
For info, since this is only a "bugfix", I'm rebasing this PR to update it + remove the "PR title templating" tentative fix that would be a subsequent PR, so we can merge the initial fix.